lib/pages/home/home_direct_list_widget.dart

```diff
@@ -46,18 +46,44 @@ class HomeDirectListWidget extends StatelessWidget {
                 itemCount: directState.channels.length,
                 itemBuilder: (context, index) {
                   final channel = directState.channels[index];
-                  return HomeChannelTile(
-                    onHomeChannelTileClick: () =>
-                        NavigatorService.instance.navigate(
-                      channelId: channel.id,
-                      workspaceId: channel.workspaceId,
-                    ),
-                    title: channel.name,
-                    name: channel.lastMessage?.senderName,
-                    content: channel.lastMessage?.body,
-                    imageUrl: channel.icon,
-                    dateTime: channel.lastActivity,
-                    channelId: channel.id,
+
+                  return FutureBuilder(
+                    //  initialData: Account init,
+                    future: Get.find<ChannelsCubit>()
+                        .fetchMembers(channel: channel)
+                        .then((value) => value.first),
```

Comment:
why value.first, what if we have 10 users in direct?

Reply:
I don't get it, how can we have several users in direct if this is a direct chat with a user?

Reply:
That's just the way the backend works, we just have to support it